mne_icalabel/tests/test_features.py

```diff
@@ -0,0 +1,354 @@
+    ncomp_eeglab = constants_eeglab['ncomp'][0, 0]
+    nfreqs_eeglab = constants_eeglab['nfreqs'][0, 0]
+    n_points_eeglab = constants_eeglab['n_points'][0, 0]
+    nyquist_eeglab = constants_eeglab['nyquist'][0, 0]
+    index_eeglab = constants_eeglab['index']
+    window_eeglab = constants_eeglab['window']
+    subset_eeglab = constants_eeglab['subset']
+
+    assert ncomp == ncomp_eeglab
+    assert nfreqs == nfreqs_eeglab
+    assert n_points == n_points_eeglab
+    assert nyquist == nyquist_eeglab
+    assert np.allclose(index, index_eeglab - 1)
+
+    # window and subset are not squeezed in matlab
+    assert window_eeglab.shape[0] == 1
+    assert np.allclose(window, window_eeglab[0, :])
+
+    # for subsets, compare if the same elements are in both
+    assert subset_eeglab.shape[0] == 1
+    assert len(set(list(subset)).difference(set(list(subset_eeglab[0, :] - 1)))) == 0
+    assert len(set(list(subset_eeglab[0, :] - 1)).difference(set(list(subset)))) == 0
+
+
+def test_eeg_rpsd_compute_psdmed():
+    """Test _eeg_rpsd_compute_psdmed function."""
+    # Raw --------------------------------------------------------------------
+    raw = read_raw(raw_eeglab_path, preload=True)
+    ica = read_ica_eeglab(raw_eeglab_path)
+    icaact = compute_ica_activations(raw, ica)
+
+    # retrieve subset from eeglab
+    constants_eeglab = loadmat(psd_constants_raw_path)['constants'][0, 0]
+    assert constants_eeglab['subset'].shape[0] == 1
+    subset_eeglab = constants_eeglab['subset'][0, :] - 1
+
+    # retrieve the rest from python
+    ncomp, nfreqs, n_points, nyquist, index, window, _ = \
+        _eeg_rpsd_constants(raw, ica)
+
+    # compute psdmed
+    psdmed = _eeg_rpsd_compute_psdmed(raw, icaact, ncomp, nfreqs, n_points,
+                                      nyquist, index, window, subset_eeglab)
+
+    psdmed_eeglab = loadmat(psd_psdmed_raw_path)['psdmed']
+    assert np.allclose(psdmed, psdmed_eeglab, atol=1e-4)
+
+    # Epochs -----------------------------------------------------------------
+    epochs = read_epochs_eeglab(epo_eeglab_path)
+    ica = read_ica_eeglab(epo_eeglab_path)
+    icaact = compute_ica_activations(epochs, ica)
+
+    # retrieve subset from eeglab
+    constants_eeglab = loadmat(psd_constants_epo_path)['constants'][0, 0]
+    assert constants_eeglab['subset'].shape[0] == 1
+    subset_eeglab = constants_eeglab['subset'][0, :] - 1
+
+    # retrieve the rest from python
+    ncomp, nfreqs, n_points, nyquist, index, window, _ = \
+        _eeg_rpsd_constants(epochs, ica)
+
+    # compute psdmed
+    psdmed = _eeg_rpsd_compute_psdmed(epochs, icaact, ncomp, nfreqs, n_points,
+                                      nyquist, index, window, subset_eeglab)
+
+    psdmed_eeglab = loadmat(psd_psdmed_epo_path)['psdmed']
+    assert np.allclose(psdmed, psdmed_eeglab, atol=5e-2)
+    # TODO: investigate why the tolerance had to be brought this high for this
+    # particular case.
```

Comment:
This one is weird. It's the first time I can't get by with a tolerance around `1e-4` (or even `1e-5)`.
The weirdest part is below, lines +297 to +302 where tolerance of `1e-5` is used for the same file _after_ formatting.

Reply:
solved.